lib/editor-linter.coffee

```diff
@@ -35,6 +33,10 @@ class EditorLinter
   onDidDestroy: (callback) ->
     return @emitter.on('did-destroy', callback)
 
+  destroy: ->
+    @emitter.emit('did-destroy')
```

Comment:
Shouldn't this be at the end of the function since it is `did-destroy` not `will-destroy`?